src/pages/simulator/result/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link } from '@tanstack/react-router';
+import { SimResultChart } from 'components/simulator/result/SimResultChart';
+import { SimResultSummary } from 'components/simulator/result/SimResultSummary';
+import { useStrategyInput } from 'hooks/useStrategyInput';
+import { useSimulator } from 'components/simulator/result/SimulatorProvider';
+import { useSearch } from 'libs/routing';
+import { useEffect } from 'react';
+import { ReactComponent as IconChevronLeft } from 'assets/icons/chevron-left.svg';
+import { wait } from 'utils/helpers';
+
+export const SimulatorResultPage = () => {
+  const ctx = useSimulator();
+  const searchState = useSearch({ from: '/simulator/result' });
+  const { state } = useStrategyInput({ searchState });
+  const simulationType = 'recurring';
+
+  useEffect(() => {
+    if (!ctx.isSuccess || ctx.status === 'running' || ctx.status === 'ended') {
+      return;
+    }
+
+    wait(3000).then(() => {
```

Comment:
I am guessing this is the part that waits 3 seconds to start the animation - but an anonymous arrow function with an hardcoded number isn't a good way to help me understand it. I'd suggest a named function and a named constant for both